packages/lodestar/src/network/reqresp/encodingStrategies/sszSnappy/encode.ts

```diff
@@ -29,7 +29,7 @@ export async function* writeSszSnappyPayload<T extends RequestOrOutgoingResponse
  * Buffered Snappy writer
  */
 function encodeSszSnappy(bytes: Buffer): AsyncGenerator<Buffer> {
-  const stream = createCompressStream();
+  const stream = createCompressStream({asyncCompress: true});
```

Comment:
@dapplion @wemeetagain do we want to go with async or sync version? the sync version is 2x faster, and payloads are mostly 1 chunk (~`0.0229 ms` compress time) (the big bellatrix goerli block was 2 chunks ~`0.0562 ms` compress time).

Reply:
Imo lets go with sync. If we start dealing with larger payloads, we might revisit, selectively use async in those cases.